### ISSUE TITLE
Please support additional image file format

### ISSUE THREAD

**Issue by jonbhanson** [jonbhanson/flutter_native_splash#245]
This thread will be used to capture all requests for support of additional image file formats.  To show your desire for a particular file format on a given platform, give the associated reply below a thumbs up.  If your desired format/platform combination is not listed, add your own reply with a single format/platform combination for each reply.

**Comment by AlexandreMarkus** [comment]
@jonbhanson I think this issue should not be close since all of linked issues have :+1: . I would like the .webp format for web please

**Comment by Prn-Ice** [comment]
@jonbhanson [this package ](https://pub.dev/packages/animated_native_splash) adds support for using lottie files as splash items, might help with #96.

**Comment by sawirricardo** [comment]
where is gif?

**Comment by ChauCM** [comment]
gif please, or anyway to let user know they are waiting on a 10s initial load in web :(. Static image might make it feel like the web is freeze/stops responding.

**Comment by kevin2-cyber** [comment]
support gif

**Comment by jinxiking** [comment]
support gif thank you~

**Comment by Bungeefan** [comment]
SVG support would be very appreciated

**Comment by mohamed-benoba** [comment]
Please support .riv file format, Rive is a new animation tool that has a lot of potenioal, lots of apps are using it now like Dolingou

**Comment by OutdatedGuy** [comment]
Yo everyone 🤟🏼

I am try to implement support for gif images in splash screen for the `web` platform in the PR #547

Can you guys try testing it out and tell if it works properly?

**Comment by FullstackWEB-developer** [comment]
Hi @jonbhanson 
Thanks for the useful package.
Is there any other way to create a splash page like this to use animations for the splash screen?

https://github.com/jonbhanson/flutter_native_splash/issues/123#issuecomment-775153422

I implemented animation in this way, but I want to implement it with only `flutter_native_splash` package.
Having one page for splash is unnatural.

**Comment by laterdayi** [comment]
It has been two years, is there any update progress?
@jonbhanson

**Comment by jonbhanson** [comment]
Yes, GIF/Lottie has the most votes above and @OutdatedGuy added GIF support with #547.  I don't have time at the moment to add any other formats, but PRs are always welcome.

**Comment by laterdayi** [comment]
The time is expected to be released
  @jonbhanson

**Comment by jonbhanson** [comment]
> The time is expected to be released @jonbhanson

It was release in May.  Just search the readme for "GIF".

**Comment by laterdayi** [comment]
@jonbhanson     thank you help ,I see that gif only supports web platform, whether it supports Android ios
![图片](https://github.com/jonbhanson/flutter_native_splash/assets/122137647/39a93364-44dc-49fc-a095-a3bc9522bb06)

**Comment by vIceBerg** [comment]
I'm adding my voice to the SVG support request.

There's a library that converts a SVG to an image:
https://pub.dev/packages/flutter_svg

Maybe this package could use this

**Comment by Zhamshid** [comment]
Is it support now .gif  for Android and iOS?

**Comment by md-rifatkhan** [comment]
any update?

**Comment by jonbhanson** [comment]
I do not have bandwidth to add additional image formats.  They will have have to come as PRs from others.

**Comment by PropFault** [comment]
Please support SVG! Would be great for high-res files especially on older devices :)

**Comment by phamquoctrongdev** [comment]
SVG format still not support?
We need it.
Thank you!

**Comment by HudsonAfonso** [comment]
SVG +1
Lottie +1

**Comment by DarkDrim** [comment]
SVG +1

**Comment by shadowfoxish** [comment]
SVG + 1
